logging/runtime/src/main/java/io/quarkiverse/googlecloudservices/logging/runtime/LoggingConfiguration.java

```diff
@@ -26,6 +27,12 @@ public class LoggingConfiguration {
     @ConfigItem(defaultValue = "true")
     public boolean enabled;
 
+    /**
+     * Enable or disable default Quarkus console logging.
+     */
+    @ConfigItem
+    public boolean enableConsoleLogging;
```

Comment:
This config property is not used anywhere

Reply:
it is used, inside `LoggingBuildSteps`, where we configure if we want to disable Quarkus console logging

Reply:
Yes but it is not used in runtime code so it seems useless, the logTarget is enough IMHO

Reply:
It is not useless because it disables console logging with which we avoid duplicated logs. This GCP extension is currently registering another log handler, which causes us to have 2 logging handlers and 2 log outputs. With this option, we have the option to disable this by default so that user does not have to disable them manually (but they still have the option to keep console logging if they want).

Reply:
Just understood what you did!

If someone want to disable console login, he can use the `quarkus.log.console.enable` property, I don't see the point of providing another config property that will set this one for him.